update readme to correctly bootstrap AWS project (#820)
openfun/marsha: docs/env.md

#### TF_VAR_migrations

A list of migrations to execute by the migration lambda without the extension (`.js`). List of migrations are available in `src/aws/lambda-migrate/src/migrations`.
To execute the migration `0001_encode_timed_text_tracks.js` set this variable with `0001_encode_timed_text_tracks`

Example: ["0001_encode_timed_text_tracks"]

> - Type: string list

Reviewer: ```suggestion
- Type: list of strings
```

- Required: No
- Default: []

#### TF_VAR_lambda_image_name

> The [ECR](https://aws.amazon.com/ecr/) image name you want to use. All lambda are built in one docker image and hosted on this repo

Reviewer: ```suggestion
The [ECR](https://aws.amazon.com/ecr/) image name you want to use. All lambdas are built in one docker image and hosted in this registry.
```

Author: No we create a repository: https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecr_repository

Reviewer: oh ok. Then write "repository" in full?

Author: Yes you are right

- Type: string
- Required: Yes
- Default: None

#### TF_VAR_lambda_image_tag

> The lambda docker image tag to used by all the lambdas

Reviewer: ```suggestion
The lambda docker image tag used by all the lambdas
```

- Type: string
- Required: Yes
- Default: None

#### TF_VAR_ecr_lambda_marsha_arn

> The [arn](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) associated to the ECR image name.

Reviewer: ```suggestion
The [arn](https://docs.aws.amazon.com/general/latest/gr/aws-arns-and-namespaces.html) associated with the ECR image name.
```
